ui-components/dialog.md

```diff
@@ -136,15 +136,15 @@ The Help plugin's dialog is an example of a tab panel dialog.
 
 ### Footer buttons
 
-A **button** is a clickable component that can contain text or an icon. There are two types of buttons (primary and secondary buttons), though the only difference is that they are styled differently. Primary buttons are intended to stand out. The color will depend on the chosen [skin]({{site.baseurl}}/general-configuration-guide/customize-ui/#skins).
+A **button** is a clickable component that can contain text or an icon. There are three types of buttons (primary, secondary and menu buttons). Primary and secondary buttons will perform an action when clicked, however they are styled differently. Primary buttons are intended to stand out. The color will depend on the chosen [skin]({{site.baseurl}}/general-configuration-guide/customize-ui/#skins). Menu buttons instead of performing an action will open a menu with more options when clicked.
```

Comment:
"Menu buttons instead of performing an action will open a menu with more options when clicked." -> "Menu buttons will open a menu with more options when clicked, instead of performing an action."?